<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenJournal</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QFrame" name="productivityPage">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="3">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>170</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0" colspan="4">
         <widget class="Editor" name="entry">
          <property name="cursor" stdset="0">
           <cursorShape>IBeamCursor</cursorShape>
          </property>
          <property name="acceptDrops">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QLabel" name="date">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string notr="true">Date xx Month xxx</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>151</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="calendarPage">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="handleWidth">
           <number>20</number>
          </property>
          <widget class="QCalendarWidget" name="calendar">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="firstDayOfWeek">
            <enum>Qt::Monday</enum>
           </property>
           <property name="gridVisible">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QWebEngineView" name="preview" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>2</verstretch>
            </sizepolicy>
           </property>
           <property name="url" stdset="0">
            <url>
             <string>about:blank</string>
            </url>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>723</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;Journal</string>
    </property>
    <widget class="QMenu" name="menu_Advanced">
     <property name="title">
      <string>&amp;Advanced</string>
     </property>
     <addaction name="actionConnect_planner"/>
     <addaction name="actionCloud"/>
     <addaction name="actionBackup"/>
    </widget>
    <action name=""/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionNew_planner"/>
    <addaction name="actionOpen_planner"/>
    <addaction name="actionExport_planner"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="menu_Advanced"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>Language</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuStyle">
     <property name="title">
      <string>Style</string>
     </property>
    </widget>
    <addaction name="menuLanguage"/>
    <addaction name="menuStyle"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionSupport"/>
    <addaction name="actionAboutQt"/>
    <addaction name="separator"/>
    <addaction name="actionDocumentation"/>
    <addaction name="separator"/>
    <addaction name="actionSource"/>
    <addaction name="actionReport"/>
    <addaction name="actionAsk"/>
    <addaction name="separator"/>
    <addaction name="actionCheckRel"/>
   </widget>
   <widget class="QMenu" name="menuEntry">
    <property name="title">
     <string>&amp;Entry</string>
    </property>
    <addaction name="actionExport_current"/>
    <addaction name="actionSave_current"/>
    <addaction name="separator"/>
    <addaction name="actionImport_entry"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEntry"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNew_planner">
   <property name="text">
    <string>&amp;New journal</string>
   </property>
   <property name="toolTip">
    <string>Create a new local journal.</string>
   </property>
  </action>
  <action name="actionOpen_planner">
   <property name="text">
    <string>&amp;Open journal</string>
   </property>
   <property name="toolTip">
    <string>Open local journal</string>
   </property>
  </action>
  <action name="actionSave_planner">
   <property name="text">
    <string>Save planner</string>
   </property>
  </action>
  <action name="actionExport_planner">
   <property name="text">
    <string>&amp;Export journal</string>
   </property>
   <property name="toolTip">
    <string>Export the current journal in pdf</string>
   </property>
  </action>
  <action name="actionBackup">
   <property name="text">
    <string>&amp;Backup journal</string>
   </property>
   <property name="toolTip">
    <string>Make a temporary backup for local journal</string>
   </property>
  </action>
  <action name="actionAddAlarm">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About OpenJournal</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionConnect_planner">
   <property name="text">
    <string>&amp;Connect to remote journal</string>
   </property>
   <property name="toolTip">
    <string>Connect to a remote journal</string>
   </property>
  </action>
  <action name="actionSource">
   <property name="text">
    <string>Show me the code</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionFrench">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>French</string>
   </property>
  </action>
  <action name="actionExport_current">
   <property name="text">
    <string>&amp;Export current entry as Pdf</string>
   </property>
  </action>
  <action name="actionSave_current">
   <property name="text">
    <string>&amp;Save current entry as Markdown</string>
   </property>
  </action>
  <action name="actionImport_entry">
   <property name="text">
    <string>&amp;Import Markdown file</string>
   </property>
  </action>
  <action name="actionReport">
   <property name="text">
    <string>Report an Issue</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>User Manual</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionCloud">
   <property name="text">
    <string>&amp;Cloud connection</string>
   </property>
  </action>
  <action name="actionAsk">
   <property name="text">
    <string>Ask a Question</string>
   </property>
  </action>
  <action name="actionCheckRel">
   <property name="text">
    <string>Check for new release</string>
   </property>
  </action>
  <action name="actionSupport">
   <property name="text">
    <string>Support OpenJournal</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
  <customwidget>
   <class>Editor</class>
   <extends>QPlainTextEdit</extends>
   <header>editor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
